src/NzbDrone.Core/Notifications/Notifiarr/NotifiarrSettings.cs

```diff
@@ -17,8 +17,15 @@ public class NotifiarrSettings : IProviderConfig
     {
         private static readonly NotifiarrSettingsValidator Validator = new NotifiarrSettingsValidator();
 
+        public NotifiarrSettings()
+        {
+            Environment = 1;
```

Comment:
Use `(int)NotifierEnvironment.Live` so that this is easier to read in code and doesn't rely on maintaining explicit number assignments on the enum.